adapter/oss/app_client.go

```diff
@@ -0,0 +1,103 @@
+package oss
+
+import (
+	"context"
+	"fmt"
+	"github.com/fnproject/cli/adapter"
+	oss "github.com/fnproject/fn_go/clientv2"
+	apiapps "github.com/fnproject/fn_go/clientv2/apps"
+	"github.com/fnproject/fn_go/modelsv2"
+	"os"
+)
+
+type AppClient struct {
+	client *oss.Fn
+}
+
+func (a AppClient) CreateApp(app *adapter.App) (*adapter.App, error) {
+	resp, err := a.client.Apps.CreateApp(&apiapps.CreateAppParams{
+		Context: context.Background(),
```

Comment:
This looks like a reasonable to create the context.

Reply:
This is just a refactor of the code already present. We are not looking to make changes to or introduce calling cancel, configuring timeouts, etc